apps/communications/dialer/test/marionette/keypad_test.js

```diff
@@ -7,7 +7,8 @@ var ReflowHelper =
 marionette('Dialer > Keypad', function() {
   var assert = require('assert');
 
-  var client = marionette.client({ profile: Dialer.config });
+  var client = marionette.client({ profile: Dialer.config,
+  desiredCapabilities: { raisesAccessibilityExceptions: false } });
```

Comment:
Nit: We can put the desired capabilities inside Dialer.config. I currently don't see any case where we don't want to hide this exception. Btw, you don't need to make this change yourself, it'll be done in [bug 1232680](https://bugzilla.mozilla.org/show_bug.cgi?id=1232680)